### DISCUSSION TITLE
Any plans to bring this to React Native?

### DISCUSSION THREAD

**Discussion by charlestbell** [testing-library/jest-dom#419] [Ideas]
Any plans to bring this to React Native?

**Comment by gnapse** [comment]
Does React Native have a DOM? This library is very focused on the DOM as the target for where and how to test things. I have zero experience with React Native, so I'm not sure how transferable this all is as is, or how to make it work on it, if it's technically possible.

Is this something you can clear up? And if it's all feasible, is this something you can contribute yourself?

**Comment by nickserv** [comment]
I know this is old, but for historical reference, the solution is to use [`@testing-library/jest-native`](https://github.com/testing-library/jest-native).

**Reply by charlestbell** [reply]
That's what I'm using. It's just missing things like ".toBeInTheDocument" and such. It works, it's just not as human-friendly
